crates/server/src/handle.rs

```diff
@@ -109,12 +121,16 @@ pub async fn handle(mut req: Request<Body>) -> impl IntoResponse {
             )),
         },
         (Some("_tag"), Some(tag), prop @ (None | Some("tree"))) => {
-            let tag = tag.parse::<TagName>().map_err(|e| {
-                (
-                    StatusCode::BAD_REQUEST,
-                    format!("Failed to parse tag name: {e}"),
-                )
-            })?;
+            let tag = tag
+                .trim_start_matches('/')
+                .trim_end_matches('/')
```

Comment:
This is redundant due to the `split` and the `filter`. You'll never get slashes here.
BTW, just FYI, `trim_matches` trims from both sides